nvm.sh

```diff
@@ -79,11 +79,18 @@ nvm_curl_libz_support() {
   curl -V 2>/dev/null | nvm_grep "^Features:" | nvm_grep -q "libz"
 }
 
+nvm_curl_use_compression() {
+  if nvm_curl_libz_support && nvm_version_greater_than_or_equal_to nvm_curl_version 7.21.0; then
+    return 0;
```

Comment:
The body of the function can just be `nvm_curl_libz_support && nvm_version_greater_than_or_equal_to nvm_curl_version 7.21.0`